### DISCUSSION TITLE
How to hide the titlebar?

### DISCUSSION THREAD

**Discussion by glepnir** [fltk-rs/fltk-rs#1261] [Q&A] [answered]
Hi folks .how to hide the title bar?

![image](https://user-images.githubusercontent.com/41671631/181192539-3b272e2a-f7a7-4d20-a540-f7e7005adbf8.png)

**Comment by MoAlyousef** [comment, marked as answer]
Hi
You can try: `win.set_border(false)`

**Reply by glepnir** [reply]
make sense. Thanks for your work @MoAlyousef  :)

**Reply by MoAlyousef** [reply]
Most welcome :)
